extensions/typescript-language-features/src/languageFeatures/tsconfig.ts

```diff
@@ -154,7 +154,7 @@ async function getTsconfigPath(baseDirUri: vscode.Uri, pathValue: string): Promi
 			return absolutePath;
 		}
 		return absolutePath.with({
-			path: `${absolutePath.path}.json`
+			path: `${absolutePath.path}/tsconfig.json`
```

Comment:
From https://github.com/microsoft/TypeScript/blob/febfd442cdba343771f478cf433b0892f213ad2f/src/compiler/commandLineParser.ts#L3005 I think we need to check both `PATH.json` and `PATH/tsconfig.json`, at least when resolving the path in `extends`

Reply:
Updated. From my own testing and looking at typescript documentation, extends and references are different in resolution. I have added a `linkType` parameter to the navigation command to resolve this, though we can also create a separate command if necessary.